src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -8,31 +14,97 @@ import messages from '../../Messages';
 /**
  * System inventory card for showing system inventory and status.
  */
-const SystemInventoryCard = () => {
+const SystemInventoryCard = ({
+    fetchInventory, inventoryFetchStatus, inventorySummary,
+    fetchInventoryStale, inventoryStaleFetchStatus, inventoryStaleSummary,
+    fetchInventoryWarning, inventoryWarningFetchStatus, inventoryWarningSummary,
+    fetchInventoryTotal, inventoryTotalFetchStatus, inventoryTotalSummary
+}) => {
+
+    useEffect(() => {
+        fetchInventoryTotal();
+        fetchInventory();
+        fetchInventoryStale();
+        fetchInventoryWarning();
+    }, [fetchInventoryTotal,
+        fetchInventory,
+        fetchInventoryStale,
+        fetchInventoryWarning]
+    );
 
     const intl = useIntl();
 
     return <TemplateCard appName='SystemInventory'>
         <TemplateCardHeader subtitle={ intl.formatMessage(messages.systemInventoryTitle) }/>
         <TemplateCardBody isFilled={ false }>
-            <NumberDescription
-                data="100000"
-                dataSize="lg"
-                percentageData={ intl.formatMessage(messages.systemInventoryPercentageData) }
-                linkDescription={ intl.formatMessage(messages.systemInventoryDescription) }
-            />
-            <IconInline
-                message={ intl.formatMessage(messages.systemInventoryWarning) }
-                state="warning"
-                systemInventory="true"
-            />
-            <IconInline
-                message={ intl.formatMessage(messages.systemInventoryDanger) }
-                state="critical"
-                systemInventory="true"
-            />
+            { inventoryFetchStatus === 'fulfilled' && inventoryTotalFetchStatus === 'fulfilled' &&
+                <NumberDescription
+                    data={ inventorySummary.total || 0 }
+                    dataSize="lg"
+                    percentageData={ intl.formatMessage(messages.systemInventoryPercentageData,
+                        { count: Math.floor((inventorySummary.total / inventoryTotalSummary.total) * 100) || 0 }
+                    ) }
+                    linkDescription={ intl.formatMessage(messages.systemInventoryDescription) }
+                    link='./insights/inventory'
```

Comment:
woaaaahhhhh we can link this way, sans UI_BASE  constant? n it'll work beta etc?

Reply:
Yeah, Karel told me this the other day in another PR elsewhere